### PR TITLE
I added new loss.py file.

### PR DESCRIPTION
**Pull request by hi-sushanta**
At that time, this file had two loss functions: MSE and MAE. However, it is also capable of storing different types of loss functions in a single file, similar to the PyTorch type framework.

Now, see what this file looks like:

Screenshot from 2023-10-16 20-13-00

### CODE REVIEW

**Comment by g1y5x3** [comment]
These loss are simple enough that I think you can just include in any of the scripts that uses them instead of going to the core library.

**Comment by hi-sushanta** [comment]
My main motive is to create a separate file to handle all types of loss calculation, similar to the PyTorch loss.py file.

**Comment by geohot** [comment]
Closing per what @g1y5x3 said.
